cyberark-secrets-provider-for-kubernetes/ci/test-values-template.yaml

```diff
@@ -0,0 +1,25 @@
+# K8s permission defaults
```

Comment:
Remove from this PR.
this should be part of the integration tests PR.

Reply:
@abrahamko don't understand your comment here. Why?